### ISSUE TITLE
MSID: 89319 Version: 1 DOI: 2023.04.03.535400

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#885]
[PLACE MANUSCRIPT AND EDITOR DETAILS HERE WHEN AVAILABLE] See step 3

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [ ] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=89319)

or (only one should be ticked. remove other from description.)

- [ ] Confirmed with Ted at bioRxiv that MECA archive is available

Send the following email to Ted and wait for his reply.

```
Hi Ted,

Please can you prepare the preprint MECA for 10.1101/2023.04.03.535400

Thanks
```

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/89319/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [ ] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.04.03.535400)
- [ ] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [ ] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/89319/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/89319)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/89319
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/89319
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/89319/v1/89319-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/89319)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by fred-atherden** [comment]
The details at the top of this ticket use the msid `89947`, @KJT08 is that a mistake, and if so, are the other details all still correct or do they need updating as well?

For 89319, the image quality of the graphical abstract is really poor and broken:

![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/7cc9bb76-d911-4081-bb78-8c5d1bf32b7a)


It's the same [in the PDF at bioRxiv](https://www.biorxiv.org/content/10.1101/2023.04.03.535400v2.full.pdf):


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/4e8d08d3-93e0-4aa5-821a-7dca733730cd)

But oddly not the same in the PDF in eJP. We also have the original jpeg image, showing that in addition to poor quality, a lot of the cell images are missing:



![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/ff09a30d-322c-40bd-92b0-eb4fc73a7411)



@acollings (not urgent - for when you are back), I'm not sure what's gone on here but I suspect we need to go back to the authors and ask them to update the version on bioRxiv before we can proceed with this one. Or would that cause problematic confusion at this stage?

**Comment by KJT08** [comment]
@fred-atherden Apologies, the msid 89947 is wrong it should be 89319, all the other details are correct.

**Comment by fred-atherden** [comment]
Great - thanks Karen.

**Comment by acollings** [comment]
Thanks @fred-atherden. If it's only the graphical abstract, and as it was posted like this by the authors, I'd be comfortable proceeding. However, I'd be happy to ask the authors to update their preprint, if it would help, so just let me know! Cheers, Andy

**Comment by acollings** [comment]
Will do, @fred-atherden!

**Comment by fred-atherden** [comment]
Many thanks!

**Comment by acollings** [comment]
From the authors: I confirm that I have uploaded a revised version of the preprint, improving the quality of the graphical abstract.

@fred-atherden FYI : I guess we need to wait for V3 to appear (I can't see it yet!)

**Comment by fred-atherden** [comment]
Nice one - thanks Andy. I'll check in with this one in the next day or two

**Comment by fred-atherden** [comment]
V3 full text is available but the MECA package isn't yet:

`curl 'https://api.biorxiv.org/meca_index/elife/all/2023.04.03.535400' | jq`

```json
{
  "status": [
    {
      "count": 1,
      "messages": "ok"
    }
  ],
  "results": [
    {
      "msid": "535400",
      "tdm_doi": "10.1101/2023.04.03.535400",
      "ms_version": "2",
      "filedate": "2023-05-15",
      "tdm_path": "s3://transfers-elife/biorxiv_Current_Content/May_2023/16_May_23_Batch_1613/9cc0713b-6c78-1014-bbfa-a46f51a53a35.meca",
      "transfer_type": "api"
    }
  ]
}
```

If it's not available tomorrow, I'll contact bioRxiv
